actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -243,8 +243,13 @@ def content_type=(content_type)
     end
 
     # Content type of response.
-    # It returns just MIME type and does NOT contain charset part.
     def content_type
+      type = super
+      type&.empty? ? nil : type
```

Comment:
This line is a bit hard to understand. Is it not enough `super.presence`?

Reply:
:+1:  I fixed with 22274d320e36a4750fd83b096fb3a490c96559d6. Thanks!